src/twisted/web/_http2.py

```diff
@@ -103,11 +103,23 @@ class H2Connection(Protocol, TimeoutMixin):
     @ivar _sender: A handle to the data-sending loop, allowing it to be
         terminated if needed.
     @type _sender: L{twisted.internet.task.LoopingCall}
+
+    @ivar abortTimeout: The number of seconds to wait after we attempt to shut
+    the transport down cleanly to give up and forcibly terminate it. This
+    is only used when we time a connection out, to prevent errors causing
+    the FD to get leaked. If this is L{None}, we will wait forever.
```

Comment:
It might also be helpful here to note that this is the timeout _from_ the time that the other timeout has fired until `abortConnection`, not the amount of time from the start.